dart/external/imgui/CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ include_directories(
   PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}"
 )
 
+if(CMAKE_COMPILER_IS_GNUCXX)
+  target_compile_options(dart-external-imgui PRIVATE -w)
```

Comment:
Nit: Let's use the target name variable (i.e. `${target_name}`) instead of `dart-external-imgui`.